### ISSUE TITLE
Button icons design is not intuitive and tooltips for Console sessions are missing

### ISSUE THREAD

**Issue by rodrigosf672** [posit-dev/positron#13540]
<img width="528" height="356" alt="Image" src="https://github.com/user-attachments/assets/b37151f1-52f2-4411-908e-07d57a7c6480" />

#### Positron and OS details:

```
Version: Positron Version: 2026.06.0 build 68
Code - OSS Version: 1.111.0
Commit: 44f5ce88c84eae2092c3e8b039c7560415730dcd
Date: 2026-05-12T17:43:59-04:00
Commit: 39.6.0
Date: 142.0.7444.265
Electron: 22.22.0
ElectronBuildId: 14.2.231.22-electron.0
Chromium: Darwin arm64 25.4.0
Node.js: undefined
V8: undefined
OS: undefined
```

#### Session details:

`any`

## Describe the issue:
Button icons design is not intuitive, which makes it confusing for users, especially due to missing tooltips for Console sessions. 
- Third button looks like it will somehow "copy" the console session.
- Fourth button resembles a "stop" button that can stop the console session.

## Steps to reproduce the issue:

Check the console sessions buttons. See that tooltips are missing and that the two right buttons have misleading design icons. 

## Expected or desired behavior:
Tooltips and icons that better represent the actions. 

- Open the Developer Tools console by running the `Developer: Toggle Developer Tools` command from the Command Palette
- Open the relevant Output Channel logs by utilizing this guide: https://positron.posit.co/troubleshooting.html

-->

**Comment by juliasilge** [comment]
Related to https://github.com/posit-dev/positron/issues/2228
